libgit.el

```diff
@@ -65,7 +72,7 @@ On successful exit, pass control on to the build step."
 On successful exit, pass control on to the load step."
   (let ((default-directory libgit--build-dir))
     (set-process-sentinel
-     (start-process "libgit-cmake" "*libgit build*" "make")
+     (start-process "libgit-cmake" "*libgit build*" "cmake" "--build" ".")
```

Comment:
Will `cmake --build .` fallback to calling `make` if we didn't specify a generator above?

Reply:
Cmake will default to the platforms default. 

It could also make sense to pass -S to cmake to avoid changing the directory to the source directory.

Reply:
Thanks for the feedback, but I have decided to close this until @vconcat gets back to this and makes the requested change and provides all the necessary information in the commit message.